h2/config.py

```diff
@@ -6,6 +6,27 @@
 Objects for controlling the configuration of the HTTP/2 stack.
 """
 
+import weakref
+
+
+class _BooleanConfigOption(object):
+    """
+    Descriptor for handling a boolean config option.  This will block
+    attempts to set boolean config options to non-bools.
+    """
+    def __init__(self, name, default):
+        self.name = name
+        self.default = default
```

Comment:
I suppose not. Actually, given that we supply the defaults to the constructor of `H2Configuration`, this is really just duplicating information. I'll remove it.